### PR TITLE
Add event-model

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

**Comment by ericdill** [comment]
Any idea why AppVeyor failed on python 3.5? Sorry if this is a known issue and I'm just not seeing it

**Comment by jakirkham** [comment]
No clue. I don't think this is universally the case and we have put a hold on any changes to existing pinnings. Does this build on Python 3.5 on Windows? 😕

**Comment by ericdill** [comment]
> Does this build on Python 3.5 on Windows?

I honestly have no idea... I have never tried before, but I can't think of why it wouldn't.

**Comment by jakirkham** [comment]
See this [comment]( https://github.com/conda-forge/staged-recipes/issues/666#issuecomment-221125814 ) for a workaround.

File: recipes/event-model/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        - python
+        - enum34  # [py2k]
+
+test:
+    imports:
+        - event_model
+#    requires:
+#        - pytest
+#    commands:
+#        - py.test {{ environ.SRC_DIR }}
```
Comment (ocefpaf) [inline recipes/event-model/meta.yaml]: Do you plan on using this at some point? If not let's just remove the commented line.

**Comment by ocefpaf** [comment]
@ericdill this is good to go. Should I merge?

**Comment by ericdill** [comment]
@ocefpaf good catch on the commented tests. This is good to go, thanks

**Pull request merged**